tikv-client/src/main/java/com/pingcap/tikv/codec/MyDecimal.java

```diff
@@ -87,7 +120,7 @@ public int frac() {
    * @param value an double value
    */
   public void fromDecimal(double value) {
-    String s = Double.toString(value);
+    String s = value + "";
```

Comment:
It seems that science notation will only occur when you change this line. So is it impossible to meet the assumptions that this science notation bug really exists? Please confirm.